### ISSUE TITLE
N64 ROM Hanging on Load in 2.4.2 and 2.3.1

### ISSUE THREAD

**Issue by nerezzaconvoca** [TASVideos/BizHawk#2167]
### Summary
When loading a *.z64 ROM, BizHawk hangs at a black screen with only one line appearing in the log.  The same results occurs when drag and drop starting, starting from a menu, or command line.  This is case in 2.3.1 and the latest Windows binary.  I use 2.3.1 normally for Crowd control but tried in the latest binary just in case.  I am running Windows 10.

### Host env.
[//]: #
- BizHawk 2.3.1; Win10 Pro 1909; Intel
- BizHawk 2.4.2; Win10 Pro 1909; Intel

[//]: # "(screenshots, if applicable)"

<img width="818" alt="Annotation 2020-06-23 150512" src="https://user-images.githubusercontent.com/67344321/85471996-1dd4d300-b566-11ea-9f9f-f06b2ea27ff4.png">

**Comment by YoshiRulz** [comment]
Try changing `Config` > `Display...` > `Display Method` and restarting EmuHawk.

**Comment by nerezzaconvoca** [comment]
Same result, including Log window output.  Interestingly, if I choose OpenGL and restart, GDI+ plus is selected.  I can't seem to keep OpenGL selected.

For reference Direct3D9 (with alternate on and off) all have the same effect.

**Comment by YoshiRulz** [comment]
If you can't see anything with GDI+, I take it you don't have a dedicated graphics card? The drivers for Intel's iGPUs do not like the OpenGL-powered Mupen graphics plugins: see #1820. You might have some luck with the Rice plugin.

**Comment by nerezzaconvoca** [comment]
Oh interesting.  Correct, there is no dedicated graphics card.  I did leave out an important piece of info in the original post, I was running this on Glide in Windows 7 and now it's stopped working now that I upgraded to Windows 10.  Does that change or explain anything more?

Unfortunately I can't choose a plugin because I can't get an N64 ROM to load, unless there is a command like to force the ROM to run on BizHawk with Rice?
